### ISSUE TITLE
Setting ODBC DBMS Version to Cdn Version

### ISSUE THREAD

**Issue by simplicbe** [simplic-systems/simplic-cdn-odbc#20]
@jts93 Should we set the dbms version to the simplic cdn version? 

```c++
m_info.addField(SQL_DBMS_VER, new StringInfoField("00.00.0000"));
```

**Comment by simplicbe** [comment]
@jts93 any news here?

**Comment by jts93** [comment]
Setting this to the Simplic.CDN version would make sense. Ideally, this would be done by calling a Simplic.CDN API that returns that version string. See https://github.com/simplic-systems/simplic-cdn/issues/8

**Comment by simplicbe** [comment]
But the Simplic.CDN version might be different from the ODBC Driver version, or not?

**Comment by jts93** [comment]
Yes, that's why we need the API. The `SQL_DBMS_VER` request type is used to request the version of the DBMS (=Simplic.CDN), not the driver.

**Comment by simplicbe** [comment]
Ah, got it. I'll implement it.

**Comment by simplicbe** [comment]
Done: https://github.com/simplic-systems/simplic-cdn/issues/8
